osf/utils/datetime_aware_jsonfield.py

```diff
@@ -12,13 +12,23 @@
 from psycopg2.extras import Json
 
 
+class NaiveDatetimeException(BaseException):
```

Comment:
Custom exceptions should inherit from `Exception` not `BaseException`. Source: https://docs.python.org/3/library/exceptions.html?highlight=baseexception#BaseException